src/viur/core/request.py

```diff
@@ -491,8 +491,8 @@ def test(a: List[str])
 
     def findAndCall(self, path: str, *args, **kwargs) -> None:
         """
-            Does the actual work of sanitizing the parameter, determine which @exposed (or @internalExposed) function
-            to call (and with witch parameters)
+            Does the actual work of sanitizing the parameter, determine which exposed-function to call
+            (and with which parameters)
```

Comment:
```suggestion
            (and with wizard parameters)
```
? 🧙🪄

Reply:
No, it aren't wizard parameters. The docstring is generally wrong, I've just fixed a typo.